pkg/coscheduling/core/core.go

```diff
@@ -209,6 +226,19 @@ func (pgMgr *PodGroupManager) Permit(ctx context.Context, pod *corev1.Pod) Statu
 	if int32(assigned)+1 >= pg.Spec.MinMember {
 		return Success
 	}
+
+	if assigned == 0 {
```

Comment:
Based on my understanding, when `assigned == 0`, we should and only should trigger the `activating`, then in `ActivateSiblings`, once `Activate is Ture`, we should start the trigger but now we're the opposite. And what's more, shouldn't we set the `Activate to False` in the follow up.

Reply:
> when assigned == 0, we should and only should trigger the activating, then in ActivateSiblings, once Activate is Ture, we should start the trigger but now we're the opposite.

That's the current case. Why do you think it's the opposite.

> shouldn't we set the Activate to False in the follow up.

Nope b/c every scheduling cycle comes with its own CycleState instance.

Reply:
Make sense.
/lgtm
/hold cancel